aries-site/src/pages/foundation/tshirt-sizing.mdx

```diff
@@ -2,37 +2,120 @@ import { Box, Image } from 'grommet';
 
-## What is t-shirt sizing?
-
-T-shirt sizing is the way that Grommet leverages familiar clothing sizes of "small", "medium", "large", and other ranges of sizes to drive the sizing and spacing of components. Grommet maps these t-shirt sizes to specific pixel values. The mapping and pixel value derivation is [explained below](/foundation/tshirt-sizing#the-base-unit).
-
-T-shirt sizing controls two aspects of a user interface: 
-- the [height and width](/foundation/tshirt-sizing#t-shirt-sizing-for-component-dimensions) of layout components
-- the [spacing](/foundation/tshirt-sizing#t-shirt-sizing-for-spacing-and-other-styles) within and around components (e.g. gap, margin, padding, border, etc.)
-
-When designing and developing with the HPE Design System, you should always use t-shirt sizes. If you find yourself specifying custom pixel values, this should be a warning signaling either:
-- A need to consult with your designer about why a custom value is needed. 
-- Something in the implementation can and should be simplified.
-- Or, there could be a gap in the Design System which should be reported and addressed.
-
-In nearly every case, designs and development should be using t-shirt sizes for compatibility and maintenance.
-
-## Why do we use t-shirt sizing?
-
-T-shirt sizing delivers a more unified experience for HPE customers by creating consistent user interfaces across pages and across applications.
-
-There are also benefits of t-shirt sizing for designers and developers!
-
-T-shirt sizing creates a common language between designers and developers, as well as provides common, well-defined intervals which make designing and developing more efficient.
-Using pixel values, percentages, or other sizing methods to create layouts can result in applications that are less composable and less scalable. Instead, a container of content may just have a width of "medium", for example, and Grommet will take care of the rest.
-
-Finally, t-shirt sizes enable content to responsively scale more easily and consistently. For example, a layout's padding may be set to "large" on desktop screen and gracefully adjust to "small" on mobile devices with minimal effort.
-
-## The base unit
-
-In order to create the range of sizes and their corresponding pixel values, a base unit of `24px` is used as a scalar.
-
-Below is a snippet from Grommet's [`base.js`](https://github.com/grommet/grommet/blob/master/src/js/themes/base.js) theme file where the calculations and resulting pixel values for the t-shirt sizes driving the height/width for components such as [Box](/components/box) are defined. The `base.js` theme establishes many of the defaults leveraged across Grommet applications.
+## Summary
+
+- Named size tokens (e.g. `small`, `medium`) replace raw pixel values.
+- The system governs both component dimensions and spacing (padding, gap, margin, border, radius).
+- The scale derives from two base units: 24px (original) and 16px (supplemental in v7).
```

Comment:
We jump in to talking about v7 and v6. Since we have a lot of different package offering s let's make it clear that we are referring to grommet-theme-hpe here.

Let's also try to structure the information in such a way that it will hold up long term. For example if we are on v9 or v10 of the theme, this page should still make sense to users. The way that this reads currently is that the supplemental base unit only applys to v7